tox.ini

```diff
@@ -75,3 +75,13 @@ commands=
 [pytest]
 testpaths = tests
 addopts = -v
+
+[testenv:integration-tests]
+envlist = py37
+passenv = *
+deps=
+	-rrequirements.txt
+commands_pre=
+	python -m pip install -rtest-requirements.txt
```

Comment:
Sorry to throw a wrench in the works but I think we have a security problem here.  test-requirements.txt, unlike requirements.txt, currently isn't using hash checking mode, and we're running tests in an environment where AWS credentials are available (I guess in environment variables?)

This means anyone with the ability to write to any PyPI projects in the dependency tree of our test deps can harvest our AWS credentials.

If we're going to install from test-requirements.txt in an environment which has our secrets available, can we please use hash checking mode there as well?